test/inductor/test_fp8.py

```diff
@@ -15,6 +16,26 @@
 torch.set_float32_matmul_precision("high")
 
 
+# Utility functions are copied from
+# https://github.com/pytorch-labs/float8_playground/blob/main/float8_playground/float8_utils.py.
```

Comment:
nit: maybe we can remove the link, since this repo isn't public yet? I think it's ok to not cite it in the landed version